qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -13,8 +13,9 @@
 import tempfile
 import traceback
 
+from . import util
+from . import coverage
 from authproxy import AuthServiceProxy, JSONRPCException
-from util import *
```

Comment:
In principle I agree with getting rid of cases of `import *`, but what I'd propose is to explicitly import the symbols that are used:
```python
from util import initialize_chain, start_nodes, # etc
```
Using the full name explicitly in every use is a) more typing b) generates a large diff (as @MarcoFalke already says)